### PR TITLE
 Single dish page is ready in design

### PR DESCRIPTION
**Pull request by Wajd-AlKayyali**

### CODE REVIEW

File: client/src/pages/products/[id].jsx
Comment (Wajd-AlKayyali) [inline client/src/pages/products/[id].jsx:1]: @mattshal1 
I would highly appreciate you use the same array in this pull request in your BE work.

File: client/src/app/components/Test.jsx
Comment (Sanad-Alshobaki) [inline client/src/app/components/Test.jsx:1]: why did you change the localhost??
keep it 3001/test

**Review by Sanad-Alshobaki — CHANGES_REQUESTED** [review]
reset the localhost on the test.js file
it belongs to the backend docker mongoDB

**Comment by Wajd-AlKayyali** [comment]
fixed now

**Review by Sanad-Alshobaki — APPROVED** [review]

**Pull request merged**